app/routes/frontpage/components/CompactEvents.css

```diff
@@ -24,29 +24,20 @@
   flex: 1;
 }
 
-.compactEvents li {
-  display: flex;
-  align-items: center;
-}
-
 .eventItem {
-  height: 26px;
-}
-
-.compactEvents a {
+  height: 28.8px; /* Used for the skeleton component */
```

Comment:
What does this mean?

Reply:
The height of an event item. This is never dynamic, and it's therefore "safe" to "hard code". The skeleton component needs to know the height of the component since it does not have any children content